src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts

```diff
@@ -119,6 +124,30 @@ export class SearchableDropdownComponent extends BaseComponent implements Contro
 
     registerOnTouched(): void {}
 
+    /* Check param type */
+    getLabelProperties(container: DotContainer): string | string[] {
+        let resultProps;
+
+        if (typeof this.labelPropertyName === 'object') {
```

Comment:
There is a better way to know if is an array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray